app/jobs/indexer_job.rb

```diff
@@ -0,0 +1,25 @@
+class IndexerJob < ApplicationJob
```

Comment:
NOTE: The implementation of this Job was based on the implementation suggested in the `elasticsearch-model` gem: https://github.com/elastic/elasticsearch-rails/tree/main/elasticsearch-model#asynchronous-callbacks

I would not have written this with an `operation` argument and `case` statement if I did it from scratch.